behave_pandas/parser.py

```diff
@@ -136,3 +141,17 @@ def parse_string(cell):
         return ''
     else:
         return cell
+
+
+def parse_dict(cell):
+    if cell == '' or cell == '""':
```

Comment:
not sure we need to care about `""` here -  putting `""` in a `dict` column does not make sense? `""` is just an edge case on the string columns, in case someone genuinely needs to have both `nan`s and empty strings in a column?